packages/web-twig/src/Resources/components/Breadcrumbs/__tests__/__snapshots__/breadcrumbsDefault.twig.snap.html

```diff
@@ -8,31 +8,32 @@
     <nav class="Breadcrumbs" aria-label="Breadcrumb">
       <ol>
         <li class="d-none d-tablet-flex">
-          <a aria-current="false" href="#rootUrl" class="link-primary link-underlined">Root</a>
+          <a aria-current="false" href="#rootUrl" class="link-primary link-underlined">Root</a> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewbox="0 0 24 24" fill="none" id="fa2b65b7e049d1eae33532715c70980f" aria-hidden="true">
```

Comment:
now the output is different from the one in the web package. Can we fix it/unify it?

Reply:
We can discuss this. Because I have intentionally moved the icon from the start to the end. So the change should be done in the **web** package. Please feel free to disagree with this because I do not see every consequence of this change yet. I have done this because of the logic of the BreadcrumbsItem component. It is much easier to treat it this way than it was before this change.

If you agree, I will make the change in the **web** package.

Reply:
Semantically it should be correct. So we can do this change in the web package as well.

Reply:
Ok, refactored in the **web** package as well.